README.md

```diff
@@ -70,7 +72,45 @@ public final class MyClassWithTracing {
 
 ### Hello "OpenCensus" stats events
 
-TODO
+Here's an example on defining TagKey, Measure and View, registering view, recording stats and get
+ViewData. For the complete example, see
+[here](https://github.com/census-instrumentation/opencensus-java/blob/master/examples/src/main/java/io/opencensus/examples/helloworld/QuickStart.java).
+
+```java
+public final class QuickStart {
+
+  // frontendKey allows us to break down the recorded data
+  private static final TagKey FRONTEND_KEY = TagKey.create("my.org/keys/frontend");
+
+  // videoSize will measure the size of processed videos.
+  private static final MeasureLong VIDEO_SIZE = MeasureLong.create(
+      "my.org/measure/video_size", "size of processed videos", "MBy");
+
+  // Create view to see the processed video size distribution broken down by frontend.
+  // The view has bucket boundaries (0, 256, 65536) that will group measure values into
+  // histogram buckets.
+  private static final View.Name VIDEO_SIZE_VIEW_NAME = View.Name.create("my.org/views/video_size");
+  private static final View VIDEO_SIZE_VIEW = View.create(
+      VIDEO_SIZE_VIEW_NAME,
+      "processed video size over time",
+      VIDEO_SIZE,
+      Aggregation.Distribution.create(BucketBoundaries.create(Arrays.asList(0.0, 256.0, 65536.0))),
+      Collections.singletonList(FRONTEND_KEY),
+      Cumulative.create());
+
+  public static void main(String[] args) throws InterruptedException {
+    TagContextBuilder tagContextBuilder =
+      tagger.currentBuilder().put(FRONTEND_KEY, TagValue.create("mobile-ios9.3.5"));
+    viewManager.registerView(VIDEO_SIZE_VIEW);
+    // Process video.
+    // Record the processed video size.
+    try (Scope scopedTags = tagContextBuilder.buildScoped()) {
```

Comment:
I think it would be clearer to move the code that deals with one video to a separate method, which could be called on each request, similar to the tracing example.  It would also help distinguish between code used for initialization and stats recording.

Reply:
Make sense, I split this block of code to a separate method `processVideo()`.